internal/core/command/v2/controller/http/command.go

```diff
@@ -59,3 +59,36 @@ func (cc *CommandController) CommandsByDeviceName(w http.ResponseWriter, r *http
 	// encode and send out the response
 	pkg.Encode(response, w, lc)
 }
+
+func (cc *CommandController) IssueGetCommandByName(w http.ResponseWriter, r *http.Request) {
+	lc := container.LoggingClientFrom(cc.dic.Get)
+	ctx := r.Context()
+	correlationId := correlation.FromContext(ctx)
+
+	// URL parameters
+	vars := mux.Vars(r)
+	deviceName := vars[v2.Name]
+	commandName := vars[v2.Command]
+
+	// Query params
+	pushEvent := utils.ParseQueryStringToString(r, v2.PushEvent, v2.ValueNo)
+	returnEvent := utils.ParseQueryStringToString(r, v2.ReturnEvent, v2.ValueYes)
```

Comment:
Missing handling of the user specified query parameters.
V1 handles them here: https://github.com/edgexfoundry/edgex-go/blob/d0bc3b104123489a76cc16a6f8c025e818c2e684/internal/core/command/get.go#L44

Reply:
Good catch!  After reconsideration, I don't think that command service shall handle pushEvent and returnEvent params here; instead, command service acted as proxy service shall simply pass the query parameters all the way to the device service.  I'lll adjust the codes to achieve this.

Reply:
https://github.com/edgexfoundry/go-mod-core-contracts/pull/486 is submitted, so that DeviceServiceCommandClient could consume the entire query strings instead of just pushEvent and returnEvent.  Once the PR merged, the V2 API could be updated accordingly.